web/packages/teleport/src/Discover/SelectResource/SelectResource.tsx

```diff
@@ -154,39 +236,86 @@ export function SelectResource({ onSelect }: SelectResourceProps) {
           for additional permissions.
         </Alert>
       )}
-      <FeatureHeader>
-        <FeatureHeaderTitle>Select Resource To Add</FeatureHeaderTitle>
-      </FeatureHeader>
-      <HeaderSubtitle>
-        Teleport can integrate into most, if not all, of your infrastructure.
-        Search for what resource you want to add.
-      </HeaderSubtitle>
-      <Box height="90px" width="600px">
-        <InputWrapper>
-          <StyledInput
+      {updateDiscoverPreferenceAttempt.status === 'error' && (
+        <Danger mt={5} details={updateDiscoverPreferenceAttempt.statusText}>
+          Could not update pinned resources:
+        </Danger>
+      )}
+      <Box my={3}>
+        <FeatureHeaderTitle>Enroll a New Resource</FeatureHeaderTitle>
+        <Text>
+          Teleport can integrate with most, if not all of your infrastructure.
+          Search below for resources you want to add.
```

Comment:
```suggestion
          Teleport can integrate with most, if not all, of your infrastructure.
          Search below for resources you want to add.
```
i think we need another comma there to separate the "extra" part of the sentence.